index.jsx

```diff
@@ -481,8 +494,11 @@ class Controls extends Component {
         <div>
           <button
             ref={(node) => { this.minusNode = node; }}
-            className={btnClass}
-            style={btnStyle}
+            className={[
+              btnClass ? btnClass : '',
+              minusBtnClass ? minusBtnClass : '',
+            ].join(' ')}
```

Comment:
If btnClass isnt defined, but minusBtnClass is `foo` then you'll end up with a space in front, like " foo"

```
> ['', 'foo'].join(' ')
" foo"
```

Reply:
[The space is ok](https://stackoverflow.com/questions/21439859/html5-compliant-trailing-space-in-class-attribute). It's not against any rule, in fact it's allowed.